.github/workflows/go.yml

```diff
@@ -0,0 +1,34 @@
+name: Go
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+
+  test-build:
+    name: test build
+    strategy:
+      matrix:
+        go-version: [1.13.x, 1.14.x, 1.15.x]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+    - name: Install libblkid-dev
+      run: sudo apt-get install libblkid-dev
+    - name: Test container build
+      run: |
+        ./test
```

Comment:
This line isn't about testing the container build specifically; it's just running the tests.

Reply:
(I was thinking it would make sense to split testing into two build steps with appropriate names.)